zerver/actions/invites.py

```diff
@@ -382,6 +382,7 @@ def do_get_invites_controlled_by_user(user_profile: UserProfile) -> List[Dict[st
                 ),
                 invited_as=invite.invited_as,
                 is_multiuse=True,
+                streams=list(invite.streams.values_list("id", flat=True)),
```

Comment:
I think this change can be moved to a separate preparatory commit and also you should add corresponding code in tests to check this field.

Reply:
This is what I propose to do for the test.  
Since there is no existing test for checking the contents of the invite objects, we can add a test as shown below in `zerver.tests.test_invite.InvitationsTestCase`

```py
def test_content_of_invite_objects(self) -> None:
        """
        Checks content of the invite objects returned by do_get_invites_controlled_by_user.
        """
        # there will be 2 things tested here
        ## 1. test for all fields in a email-based user invite
        ## 1. test for all fields in a multiuse invite
```

Does this sound okay?  
I will place these tests in appropriate commits.

Reply:
This is what I propose to do for the test.  
Since there is no existing test for checking the contents of the invite objects, we can add a test as shown below in `zerver.tests.test_invite.InvitationsTestCase`

```py
def test_content_of_invite_objects(self) -> None:
        """
        Checks content of the invite objects returned by do_get_invites_controlled_by_user.
        """
        # there will be 2 things tested here
        ## 1. test for all fields in a email-based user invite
        ## 1. test for all fields in a multiuse invite
```

Does this sound okay?  
I will place these tests in appropriate commits.

Reply:
Please verify if the test files are structured as desired.